ui/src/app/shared/services/topics.service.ts

```diff
@@ -190,8 +190,8 @@ export class TopicsService {
             ownerApplicationId: appInfo.id,
             subscriptionApprovalRequired: subscriptionApprovalRequired,
             description: description || null,
-            ...createParams,
             ...(topicType !== 'INTERNAL' && {
+                ...createParams,
```

Comment:
No, the `createParams` must be sent always, as they e.g. contain the number of partitions. If you want to leave out some of the entries of the Kafka Topic settings contained in the `createParams`, you will have to solve this at another position.